<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectToSever">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="animation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>动画显示曲线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送目标数据</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入服务器地址</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_cs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>87</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入待发送数据</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ca">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>181</width>
      <height>87</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>接受数据如下</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_sa">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>181</width>
      <height>87</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>接受数据如下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectToClient">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>监听端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s_send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送目标数据</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入待监听端口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ss">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>87</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入待发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_senddata">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟生成温度风速并发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_stopsend">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>温度：风速：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>621</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>101</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>输出：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端列表：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_client">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>新建客户端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_trasport">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>文件发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_trasport_sever_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>文件接受</string>
    </property>
   </widget>
   <widget class="QPushButton" name="udp_sever">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>udp连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
